cypress/support/tests.ts

```diff
@@ -55,6 +57,18 @@ Cypress.Commands.add('runApplicationsTest', (testName: string) => {
       cy.createApp({appName: appName, archiveName: gitUrl, customPaketoImage: paketobuild, instanceNum: 5, addVar: 'ui', route: customRoute, sourceType: 'Git URL'});
       cy.checkApp({appName: appName, checkVar: true, route: customRoute});
       break;
+    case 'downloadManifestAndPushApp':
+      cy.createConfiguration({configurationName: configuration});      
+      cy.createApp({appName: appName, archiveName: archive, sourceType: 'Archive', route: customRoute,  instanceNum: 2, addVar: 'ui', configurationName: configuration });   
+      cy.checkApp({appName: appName , checkConfiguration: true, route:customRoute, instanceNum: 2}); 
+      // Downloading manifest      
+      cy.downloadManifest({appName: appName});
+      // Delete app prior uploading from manifest
+      cy.deleteApp({appName: appName});
+      // Create app from manifest solely and check results
+      cy.createApp({archiveName: archive, sourceType: 'Archive', manifestName: manifest }); 
+      cy.checkApp({appName: appName , checkConfiguration: true, route: customRoute, checkVar: true, instanceNum: 2});
```

Comment:
You also have to delete the configuration you created at the beginning of the test:
```
      // Delete the created configuration
      cy.deleteConfiguration({configurationName: configuration});
```

Reply:
Ok, I was unsure about this one since the test was focused on applications. Added now